hieroplatform/src/main/java/org/hiero/sketches/SampleQuantileSketch.java

```diff
@@ -0,0 +1,81 @@
+package org.hiero.sketches;
+
+import org.hiero.dataset.api.ISketch;
+import org.hiero.table.ArrayRowOrder;
+import org.hiero.table.RecordOrder;
+import org.hiero.table.SmallTable;
+import org.hiero.table.api.IColumn;
+import org.hiero.table.api.IMembershipSet;
+import org.hiero.table.api.IRowOrder;
+import org.hiero.table.api.ITable;
+
+import javax.annotation.Nullable;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.hiero.table.ObjectArrayColumn.mergeColumns;
+
+public class SampleQuantileSketch  implements ISketch<ITable, SampleList>{
+
+    private final RecordOrder colSortOrder;
+
+    /**
+     * The rate at which we sample the data. It is set to be 5*(resolution)^2/dataSize,
+     * so we expect a table of size 5*(resolution)^2. This is chosen so that the desired accuracy in
+     * the quantiles is 1/resolution. It can be tuned by changing the size by a constant factor.
+     * We currently use resolution of 100, so the expected sample size is 50,000.
```

Comment:
The caller of this method is elsewhere, so the 100 is not hardwired. Perhaps you can say that "for a resolution of 100 the expected..."